lib/vsc/install/commontest.py

```diff
@@ -137,10 +137,17 @@ class CommonTest(TestCase):
     ]
 
     # Prospector commandline options (positional path is added automatically)
+    cfg_path = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'etc')
     PROSPECTOR_OPTIONS = [
-        '--strictness', 'medium',
+        '--profile-path', cfg_path,
+        '--profile', 'pylint_py3_checker.yaml',
```

Comment:
Maybe just specify the location full path (makes the check above that it actually exists easier)

Reply:
@boegel Done!